32blit-stm32/Src/SystemMenu/firmware_menu.cpp

```diff
@@ -39,13 +39,30 @@ enum MenuItem {
   VOLUME,
   SCREENSHOT,
   DFU,
-  BATTERY_INFO,
   SHIPPING,
   SWITCH_EXE,
   STORAGE,
+  SEPARATOR1,
+  BATTERY_INFO,
+  SEPARATOR2,
```

Comment:
The menu class doesn't care about the IDs, so the two separators don't need unique ones.

Reply:
I know. But the enum values are/were used to index the items, so there's that. I am gonna fix that too, as part of the menu overhault @Gadgetoid envisioned in one of these comments. Until then, it is like that.  :)

Reply:
Ah, right `LAST_COUNT` is still used as the size... (And the thing with `SWITCH_EXE`... which doesn't even do anything useful any more)

Reply:
We could maybe `sizeof(items) / sizeof(Item)` to figure out the size, and use the enum values as IDs only. And no indexing, lookup by ID if needed. Then we would not even need separator IDs or we could just `#define SEPARATOR -1` or something.